dj/construction/utils.py

```diff
@@ -105,4 +105,4 @@ def amenable_name(name: str) -> str:
             ret.append(LOOKUP_CHARS.get(char, "UNK"))
             cont = []
 
-    return "_".join(ret) + "_" + "".join(cont)
+    return ("_".join(ret) + "_" + "".join(cont)).strip("_")
```

Comment:
If someone wanted to have `_foo_` we would make it `foo` ?

Reply:
Hmm...good point. I think you're right, this would be a problem. @CircArgs can you confirm?

Reply:
Since this is the only remaining comment, I'm going to merge this to unblock the work in a few people's forks so that they can rebase.